modules/ROOT/pages/configure-acm.adoc

```diff
@@ -516,13 +507,11 @@ If you don't configure an approval process, self-registered users will become me
 ... Select *Approve Registration* (the approval that you created in the previous section).
 ... In the *Approval Steps* section, select *New Approval Step*.
 
-. In the *Step 1. Enter Name and Description* section:
-.. Enter `Step 1` in *Name*.
+. In the *Step 1. Enter Name and Description* section, enter `Step 1` in *Name*.
 . Select *Next*.
 
 . In the *Step 2. Specify Step Criteria* section, select *Next*.
-. In the *Step 3.  Select Assigned Approver* section:
-.. Select *Let the submitter choose the approver manually*.
+. In the *Step 3. Select Assigned Approver* section, select *Let the submitter choose the approver manually*.
 . Select *Save*.
```

Comment:
GLOBALLY review use of "select" vs. "click."
See style ref.

Reply:
Reviewed and fixed throughout the file.